crates/paimon/src/io/file_io.rs

```diff
@@ -34,7 +36,7 @@ impl FileIO {
     ///
     /// TODO: Support building Operator from HashMap via options.
     pub fn new(_: HashMap<String, String>) -> Result<Self> {
-        let op = Operator::from_config(MemoryConfig::default())
+        let op = Operator::new(Fs::default().root("/"))
```

Comment:
It's better not to hard-code fs here, but I think it's a good starting point.

Reply:
yes, I think here should construct the fileIO according to the scheme. I'd like to do it in another pr.